modules/install/pages/admin.adoc

```diff
@@ -1,24 +1,161 @@
+----
+
+* Verify that you have a user named `admin` in `/opt/EAP-8.0.0/standalone/configuration/mgmt-users.properties` and `/opt/EAP-8.0.0/domain/configuration/mgmt-users.properties` file, and the user has a hashed password.
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server ~]$ sudo cat /opt/EAP-8.0.0/standalone/configuration/mgmt-users.properties  | grep -v "#"
+admin=fa522cb75d2d2ac01fb09f2ab21e4302
+
+[vagrant@server ~]$ sudo cat /opt/EAP-8.0.0/domain/configuration/mgmt-users.properties  | grep -v "#"
+admin=fa522cb75d2d2ac01fb09f2ab21e4302
+----
+
+** Log in to the management console using web browser to http://localhost:9990/, as the `admin` user and `redhat123` as the password. You should see the home page of the JBoss EAP 8 management console as shown in the screenshot below.
++
+image::console.png[align="center"]
+
+* Create a new management user named `newadmin` and password `redhat123`.
+
+** Run the `add-user.sh` script as the jboss user and enter the values highlighted in bold as shown in below command output.
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server ~]$ sudo -u jboss /opt/EAP-8.0.0/bin/add-user.sh
+What type of user do you wish to add?
+ a) Management User (mgmt-users.properties)
+ b) Application User (application-users.properties)
+(a): **a**
+
+Enter the details of the new user to add.
+Using realm 'ManagementRealm' as discovered from the existing property files.
+Username : **newadmin**
+Password recommendations are listed below. To modify these restrictions edit the add-user.properties configuration file.
+ - The password should be different from the username
+ - The password should not be one of the following restricted values {root, admin, administrator}
+ - The password should contain at least 8 characters, 1 alphabetic character(s), 1 digit(s), 1 non-alphanumeric symbol(s)
+Password : **redhat123**
+WFLYDM0102: Password should have at least 1 non-alphanumeric symbol.
+Are you sure you want to use the password entered yes/no? **yes**
+Re-enter Password : **redhat123**
+What groups do you want this user to belong to? (Please enter a comma separated list, or leave blank for none)[  ]: **Press enter**
+About to add user 'newadmin' for realm 'ManagementRealm'
+Is this correct yes/no? **yes**
+Added user 'newadmin' to file '/opt/EAP-8.0.0/standalone/configuration/mgmt-users.properties'
+Added user 'newadmin' to file '/opt/EAP-8.0.0/domain/configuration/mgmt-users.properties'
+Added user 'newadmin' with groups  to file '/opt/EAP-8.0.0/standalone/configuration/mgmt-groups.properties'
+Added user 'newadmin' with groups  to file '/opt/EAP-8.0.0/domain/configuration/mgmt-groups.properties'
+----
+
+** Log in to the management console using web browser to http://localhost:9990/, as the `newadmin` user and `redhat123` as the password. You should see the home page of the JBoss EAP 8 management console.
+
+* Create a new application user named `appuser` and password `redhat123`.
```

Comment:
No need to show this here. We will show them to do this later in the course. We cannot show anything related to app users here. remove this section

Reply:
removed the section